### ISSUE TITLE
Docker image failing to start 

### ISSUE THREAD

**Issue by fjandrad** [jrl-umi3218/lipm_walking_controller#35]
When launching the docker image with the command
 ```xhost +local:docker
docker run -it --rm --user ayumi -e DISPLAY=${DISPLAY} -v /tmp/.X11-unix:/tmp/.X11-unix:rw \
    stephanecaron/lipm_walking_controller \
    lipm_walking --floor
```

Output is:

```
started core service [/rosout]
libGL error: No matching fbConfigs or visuals found
libGL error: failed to load driver: swrast
Could not initialize OpenGL
/usr/local/bin/lipm_walking: line 43:   114 Aborted                 (core dumped) choreonoid --start-simulation ${PROJECT}
[display.launch] is neither a launch file in package [lipm_walking_controller] nor is [lipm_walking_controller] a launch file name
The traceback for the exception was written to the log file
```

I suspect it might be related to the Nvidia Card any suggestions?

I will test the manual installation but fixing this would be nice.

**Comment by fjandrad** [comment]
I did the manual installation following [this](https://jrl-umi3218.github.io/lipm_walking_controller/doxygen/HEAD/build.html) instructions and adding also dependencies specified in [stephan-caron repo](https://github.com/stephane-caron/lipm_walking_controller)


It seems that something is almost working but I get the following error:

![image](https://user-images.githubusercontent.com/11043189/127197930-e3c3d62a-1e69-4954-a7d2-685cdac72870.png)

Note that it is my first time using the mc_rtc framework, just follow instructions of the documentation and it seems is not working. ( I am using a docker image to do all installs and running the code )

**Comment by gergondet** [comment]
Hi @fjandrad 

These errors might not be too problematic. Did you try to run RViZ to start the walking?

**Comment by fjandrad** [comment]
From my understanding that is the first thing I ran using the command `roslaunch mc_rtc_ticker display.launch`.

After that I run  `choreonoid --start-simulation sim_mc.cnoid`

This starts choreonoid and adds a marker on the rviz space. 
Does this mean is working? as a test I moved the interactive marker a bit forward but then the robot falls down
![image](https://user-images.githubusercontent.com/11043189/127391825-2a1a787f-43c4-41a1-a4b8-73d80ec89728.png).

I see the robot trying to shift its com as directed by the interactive marker, but I sort of expected the robot to walk after starting simulation or after the marker moved forward enough. I am probably missing something. Where do I insert the amount of steps I want the robot to perform or how do I make it walk?
